tests/MongoDB.Driver.Tests/Specifications/command-monitoring/BulkWriteTest.cs

```diff
@@ -26,18 +24,21 @@ public class BulkWriteTest : CrudOperationTestBase
     {
         private List<WriteModel<BsonDocument>> _requests;
         private BulkWriteOptions _options = new BulkWriteOptions();
-        private WriteConcern _writeConcern = WriteConcern.Acknowledged;
 
         protected override void Execute(IMongoCollection<BsonDocument> collection, bool async)
         {
-            var collectionWithWriteConcern = collection.WithWriteConcern(_writeConcern);
+            if (collection.Settings.WriteConcern == null)
+            {
+                collection = collection.WithWriteConcern(WriteConcern.Acknowledged);
+            }
```

Comment:
In new version of `bulkWrite` specification the option `writeConcern` was moved from `arguments` section to `operation`. Thus `writeConcern`does not get to scope of `BulkWriteTest` and is set at `TestRunner` level. From the initial creation of this class (https://github.com/mongodb/mongo-csharp-driver/commit/a551f55f0c5a0e42498a71472af11bdb48cf4612#diff-1573242464e0f21dbfad8096b4af1216R29), collections's `writeConcern` was always defaulted to `Acknowledged` but I didn't find the particular reason for setting this value, so I decided to leave it, though it does not affect the result of the test, it may change the behavior for future tests.
The possible refactoring option here is to move the setting of `writeConcern` upper along the call stack into `TestRunner` (this could be done in all of command monitoring operation test classes, like `UpdateOneTest`, `InsertOneTest`, etc.).

Reply:
Also, in case if we move assigning `writeConcern` value to the testRunner level, `writeConcern` value will be automatically set to the tests like `CountTest` and `FindTest`. If I'm not mistaken, `writeConcern` value has no sense for these tests, however, I don't think that there is any harm to set these values in tests here. Thoughts?